Mathlib/LinearAlgebra/Matrix/Orthogonal.lean

```diff
@@ -38,53 +38,52 @@ variable (A : Matrix m n α)
 
 open Matrix
 
-/-- `A.has_orthogonal_rows` means matrix `A` has orthogonal rows (with respect to
-`matrix.dot_product`). -/
-def HasOrthogonalRows [Fintype n] : Prop :=
+/-- `A.OrthogonalRows` means matrix `A` has orthogonal rows (with respect to
+`Matrix.dotProduct`). -/
+def OrthogonalRows [Fintype n] : Prop :=
   ∀ ⦃i₁ i₂⦄, i₁ ≠ i₂ → dotProduct (A i₁) (A i₂) = 0
-#align matrix.has_orthogonal_rows Matrix.HasOrthogonalRows
+#align matrix.has_orthogonal_rows Matrix.OrthogonalRows
 
-/-- `A.has_orthogonal_rows` means matrix `A` has orthogonal columns (with respect to
-`matrix.dot_product`). -/
-def HasOrthogonalCols [Fintype m] : Prop :=
-  HasOrthogonalRows Aᵀ
-#align matrix.has_orthogonal_cols Matrix.HasOrthogonalCols
+/-- `A.OrthogonalCols` means matrix `A` has orthogonal columns (with respect to
+`Matrix.dotProduct`). -/
+def OrthogonalCols [Fintype m] : Prop :=
+  OrthogonalRows Aᵀ
+#align matrix.has_orthogonal_cols Matrix.OrthogonalCols
 
 /-- `Aᵀ` has orthogonal rows iff `A` has orthogonal columns. -/
 @[simp]
-theorem transpose_hasOrthogonalRows_iff_hasOrthogonalCols [Fintype m] :
-    Aᵀ.HasOrthogonalRows ↔ A.HasOrthogonalCols :=
+theorem transpose_OrthogonalRows_iff_OrthogonalCols [Fintype m] :
```

Comment:
This should be `transpose_orthogonalRows_iff_orthogonalCols` (or similarly with `has`). The only place `Orthogonal` should be capitalized is:
* In the name of the `def`s themselves
* When used as a namespace, like

Reply:
Done.